src/windows.zig

```diff
@@ -3018,3 +3018,10 @@ pub extern "kernel32" fn GetTempPathW(
     nBufferLength: DWORD, // [in]
     lpBuffer: LPCWSTR, // [out]
 ) DWORD;
+
+pub extern "kernel32" fn GetFinalPathNameByHandleA(
```

Comment:
Please don’t add another symbol import for something we already do using the W version. Two reasons why:

- Every symbol we add makes Bun start slower.
- Our UTF16 -> WTF-8 conversion code is faster than the Windows API.